config/scheduler/global_variables.xml

```diff
@@ -2,20 +2,41 @@
 <globalvariables
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xmlns="urn:proactive:globalvariables:3.13" xsi:schemaLocation="urn:proactive:globalvariables:3.13 http://www.activeeon.com/public_content/schemas/proactive/globalvariables/3.13/globalvariables.xsd">
+        <!-- Multiple filters can be added, to apply different set of global variables to different workflows -->
         <filter>
             <select>
                 <!-- Xpath expressions are used to filter workflows affected by global variables.
                 Example: to select workflows belonging to the basic-examples bucket
                 <xpath><![CDATA[/job/genericInformation/info[@name='bucketName' and @value='basic-examples']]]></xpath>
                 -->
-                <xpath><![CDATA[.]]></xpath>
+                <!-- <xpath><![CDATA[.]]></xpath> -->
```

Comment:
can you please put all this in a separate filter ?

I would like to keep the default empty filter